PlatformFighter/Assets/Scripts/Rectangle.cs

```diff
@@ -19,14 +19,12 @@ protected override void jab()
     {
         UnityEngine.Debug.Log("rectangle is jabbing");
         anime.setAnimator(AnimeState.Jab);
-        lagTime = 0f;
```

Comment:
why are all the lag times deleted?

Reply:
the lag times are now all in the animation, at the end of all animation the attackLag function will be called, adding the lag time there